File: src/test/java/ClimbingStairsTest.java
import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.CsvSource;

import static org.junit.jupiter.api.Assertions.assertEquals;

class ClimbingStairsTest {
    @ParameterizedTest
    @CsvSource({"1,1", "2,2", "3,3", "4,5", "6,13"})
    void testClimbingStars(int numberOfStairs, int expectedResult) {
        ClimbingStairs solution = new ClimbingStairs();
        int actualResult = solution.climbStairs(numberOfStairs);
        assertEquals(expectedResult, actualResult);
    }
}

File: src/test/kotlin/ClimbingStairsTest.kt
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.params.ParameterizedTest
import org.junit.jupiter.params.provider.CsvSource

internal class ClimbingStairsTest {
    @ParameterizedTest
    @CsvSource("1,1", "2,2", "3,3", "4,5", "6,13")
    fun testClimbingStars(numberOfStairs: Int, expectedResult: Int) {
        val solution = ClimbingStairs()
        val actualResult = solution.climbStairs(numberOfStairs)
        Assertions.assertEquals(expectedResult, actualResult)
    }
}
